Content.Tests/DMProject/Tests/Preprocessor/macro_numeric_path.dm

```diff
@@ -0,0 +1,15 @@
+/obj/thing_1/dodaa
+	name = @#underscore 1 test#
```

Comment:
What does this have to do with the test?

Reply:
nothing, the erroring code was using a rawstring, and I didn't see a need to remove it.

Reply:
This should be removed to keep the test focused on the macro problem. If you want to test raw strings, a new test for it could be created under `Tests/Text/`.

Reply:
Sure thing, I'll do it now

Reply:
I meant the `/obj/thing_1/dodaa` in general

Reply:
oh, the error was occurring when a path was defined as `/obj/thing_##NUM` where `##NUM` was numeric. It didn't error on fully defined paths like `/obj/thing_1`. I was just making sure I didn't alter that behaviour.